testing.rst

```diff
@@ -974,7 +974,7 @@ Learn more
 * :doc:`/components/css_selector`
 
 .. _`PHPUnit`: https://phpunit.de/
-.. _`documentation`: https://phpunit.de/manual/current/en/
+.. _`documentation`: https://phpunit.de/documentation.html
```

Comment:
Should we rather use https://phpunit.readthedocs.io/ instead?

Reply:
IMO that's inconsistent with what we do elsewhere. We always link to the English version of docs we link to. For most users that saves one click and if you are interested in reading a localised version, it's easy to switch the language.

Reply:
Either's fine. This doesn't warrant that much of an extended discussion.